Libraries/LLM/Load.swift

```diff
@@ -54,9 +54,15 @@ public func load(
             }
         }
 
+        // per-model cleanup
+        weights = model.sanitize(weights: weights)
+
         // quantize if needed
         if let quantization = baseConfig.quantization {
-            quantizeIfNeeded(model: model, weights: weights, quantization: quantization)
+            quantize(model: model, groupSize: quantization.groupSize, bits: quantization.bits) {
+                path, module in
+                weights["\(path).scales"] != nil
```

Comment:
This is the fix for loading partially quantized models -- if they have "scales" then the layer is quantized.  We don't need to test the type of layer as the `quantize()` method will only convert layers that _can_ be converted.

Reply:
Note that this depends on https://github.com/ml-explore/mlx-swift/pull/73